src/MovieCard/MovieCard.js

```diff
@@ -50,15 +49,27 @@ class MovieCard extends Component {
         </>
       )
     } else if (this.props.user.name && this.state.submittedRating > 0) {
-      return ( 
+      return (
         <>
           <p className='movie-user-rating'>My Rating: {this.state.submittedRating}</p>
-          <button className='edit-user-rating'>Edit Rating</button>
+          <button onClick={this.editRating} className='edit-user-rating'>Edit Rating</button>
         </>
       )
     }
   }
 
+  determineIfRatingExists = () => {
+    if (this.props.user.name && this.props.userRating) {
+        this.setState({hasRating: true})
+      return (
+        <>
+        <p className='movie-user-rating'>My Rating: {this.props.userRating.rating}</p>
```

Comment:
This `<p/>` on 66 and on 56  can definitely be assigned to a variable somewhere as a refactor.